src/extractors/custom/www.gruene.de/index.js

```diff
@@ -0,0 +1,31 @@
+export const WwwGrueneDeExtractor = {
+  domain: 'www.gruene.de',
+
+  title: {
+    selectors: ['header h1'],
```

Comment:
Aim to be as general as possible with your selectors. More often than not, a simple `h1` selector for the title is the best way to go.